project-base/docker/nginx/nginx-google-cloud.conf

```diff
@@ -0,0 +1,66 @@
+upstream php-upstream {
+    server php-fpm:9000;
+}
+
+server {
+    listen 8080;
+    access_log /var/log/nginx/shopsys-framework.access.log;
+    root /var/www/html/web;
+    server_tokens off;
+
+    location ~ /\. {
+        # hide dotfiles (send to @app)
+        try_files @app @app;
+    }
+    location ~ ^/index.php$ {
+        # hide index.php (send to @app)
+        try_files @app @app;
+    }
+    location / {
+        # try to serve existing files directly, fallback to @app
+        try_files $uri @app;
```

Comment:
make sure that this will have lower priority than proxy_pass for /content for GCS-Bucket